### DISCUSSION TITLE
Is it a bug ? "isNuxtError is not defined"

### DISCUSSION THREAD

**Discussion by Vincent-HD** [nuxt/nuxt#16503] [Questions]
Hi,
I'm having a problem with Nuxt, I'm trying to check if the error is from Nuxt inside an server route, so I use `isNuxtError()` function, vscode tells me that's a valid function, because it's supposed to be a global like `createError()`, but when I run it, it's telling me that the function is undefined.
For now the only solution that I found is to import it via :
```typescript
import { isNuxtError } from "#app";
```
Try it here: https://stackblitz.com/edit/github-kj9e7s?file=server%2Fapi%2Ftest.get.ts
The code:
```typescript
export default defineEventHandler(async (event) => {
  try {
    throw createError({ statusCode: 400, message: 'Testing error' });
  } catch (e) {
    console.log('Error catched');
    if (isNuxtError(e)) {
      console.log('This is a nuxt error');
    }
  }
  return true;
});
```

I'm posting it in discussion because I'm very new to Nuxt3 so I'm unsure if it's supposed to be auto imported or a global.
And btw, is the auto import feature stable ? I heard that you can lose some typing hints when "using" it

**Comment by kotx** [comment]
Have this error as well. `isNuxtError` is showed in the VSCode+Volar tooling, but is not imported when I press enter, but also errors at runtime.

The aforementioned workaround doesn't work:
` ERROR  Vue app aliases are not allowed in server routes. [importing #app from server/api/messages/[chatId].post.ts]`
Instead I need to import like this: `import { isNuxtError } from "nuxt/app"`

**Comment by NicholasAntidormi** [comment]
I think `createError` in the server apis is the one from h3, not from `nuxt/app/composables/error` (that is used on the client). 
The ide says it is from there however if you look in `.nuxt/dev/index.mjs` you will see an import like this `import { ..., createError, ... } from 'file:///.../node_modules/h3/dist/index.mjs';`

Try to use `isError` from h3 instead
